draft-ietf-acme-acme.md

```diff
@@ -1207,6 +1153,9 @@ and MUST NOT issue a certificate with contents other than those requested.  If
 the server requires the request to be modified in a certain way, it should
 indicate the required changes using an appropriate error code and description.
 
+Servers MUST verify any identifier values that begin with the ASCII Compatible
+Encoding prefix "xn--" as defined in {{!RFC5890}} are properly encoded.
```

Comment:
Here's my parsing of it:

Servers Must verify any identifier values (that begin with the ASCII Compatible Encoding prefix "xn-- as defined in RFC5890) are properly encoded.